lib/scripts/collection_list.rb

```diff
@@ -0,0 +1,37 @@
+def collections_path(collection)
+  starting_collection_data = collection_title_id_data(collection)
+  if collection.collections.empty?
+    collection_paths = [[starting_collection_data]]
+  else
+    collection_paths = []
+  end
```

Comment:
This isn't super important, but since there isn't a case where `collection.collections.empty? == true` will lead to the `collection.collections.each ...` doing anything wouldn't it make sense to move that loop into the else block like this:

```
else
  collection.collections.each do |parent_collection|
    recursive_collection_path(
      parent_collection,
      [collection_title_id_data(parent_collection),
       starting_collection_data],
      collection_paths
    )
  end

  collection_paths
end
```

It makes it immediately clear that if `collection.collections.empty? == true` nothing else is done.